Makefile

```
@@ -36,16 +38,25 @@ rm-sequential:
 start-server: $(WS_PAR)/mets.xml
 	ocrd workspace -d $(WS_PAR)  -U $(SOCK) server start
 
-rm-parallel:
+stop-server: $(WS_PAR)/mets.xml
+	-ocrd workspace -d $(WS_PAR)  -U $(SOCK) server stop
+
+rm-parallel: stop-server
 	rm -rf $(WS_PAR)
 
-$(WS_PAR)/mets.xml:
+$(WS_PAR)/mets.xml: $(WS)
 	cp -r $(WS) $(WS_PAR)
 
+parallel-chunks: page_ranges != ocrd workspace -d $(WS_PAR) list-page -D $(NUMBER_OF_THREADS) -f comma-separated
```

```suggestion
parallel-chunks: page_ranges = $(shell ocrd workspace -d $(WS_PAR) list-page -D $(NUMBER_OF_THREADS) -f comma-separated)
```

Because the `ws-parallel` workspace might not exist at assignment time.

---

Indeed – must be lazy evaluation.